scripts/lint_naming_conventions.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
+# Script used to review jenkins jobs and report job names
+# not following conventions
+# https://github.com/rcbops/rpc-gating#naming-conventions
+
+import argparse
+import os
+import re
+import sys
+import yaml
+
+
+def parse_args():
+    description = "Parses given directories for rpc-gating naming conventions"
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument("--exclude-files",
+                        required=False,
+                        default=[],
+                        help="Comma seperated list of files to exclude")
+    parser.add_argument("--dirs",
+                        required=True,
+                        help="Comma seperated dirs to parse")
+    parser.add_argument("--parse-jjb", required=False,
+                        help="Parse jjb file for names",
+                        action='store_true')
+    return parser.parse_args()
+
+
+def parse_jjb_file(in_dir, in_file):
+    _rc = 0
+    filename = os.path.join(in_dir, in_file)
+    file_yaml = yaml.safe_load(open(filename).read())
+    for item in file_yaml:
+        if "project" in item:
```

Comment:
Note that the `yaml.safe_load` will throw an exception if it can't load the YAML (syntax error etc.)  Do you want to account for that?  Also, if the file is empty, the for loop bombs as file_yaml = `None`.  The former is probably worth handling, the latter probably much less likely.